test/general.tests.js

```diff
@@ -0,0 +1,189 @@
+import { Selector } from 'testcafe';
+
+fixture `Spicy sections`
+    .page `https://spicy-sections.glitch.me/automation-tests-1.html`
+    .beforeEach(async (t) => {
+    // stuff that should happen before each test run
+        await t.maximizeWindow();
+    })
+    .afterEach(async () => {
+        // stuff that should happen after each test run
+    });
+
+const heading = Selector("h1");
```

Comment:
Showing un-used variable. 

```suggestion
```

Reply:
we can clean that in a future commit as I missed it before merging